debian/neobotix-tools/DEBIAN/postinst

```diff
@@ -42,6 +44,24 @@ sed -i -e 's%GRUB_CMDLINE_LINUX_DEFAULT="quiet splash"%GRUB_CMDLINE_LINUX_DEFAUL
 sed -i -e 's%#GRUB_TERMINAL=console%GRUB_TERMINAL=console%g' /etc/default/grub
 update-grub
 
+# Configure vino
+stty_orig=$(stty -g)
+stty -echo
+IFS= read -p "Please enter VNC password: " -r passwd
+stty "$stty_orig"
+echo ''
+gsettings set org.gnome.Vino require-encryption false
```

Comment:
Seems the issue is that these parameters are not set...

Reply:
after I manually gave the settings it worked! (Note just the first 2)

Reply:
Yes, just checked it. `require-encryption` and `prompt-enabled` both are set to true. I think this settings need to be set from the defaults user account.

Reply:
efd37347b33ce5dcc2f2f2977635f771b1ad99c4 this resolves the issue.